examples/neuralNetwork-sequence-hand-gestures/sketch.js

```diff
@@ -0,0 +1,180 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates training a Hand Gesture classifier through ml5.neuralNetwork with sequeceClassification Task.
+ */
+
+let video;
+let handPose;
+let hands = [];
+
+let sequence = [];
+let targetLength = 30;
+
+let gestures = ["Gesture #1", "Gesture #2"];
+let counts = { "Gesture #1": 0, "Gesture #2": 0 };
+
+let state = "collection";
+let currGesture = gestures[0]; //set currGesture to gesture 1 by default
```

Comment:
(OCD-fueled comment:) The comments in your code seem to oscillate between capitalizing the first word, and not doing it, without much apparent reason. Best to figure out _a_ rule, and stick to it ;)
(And always a space after `//`, please)